docs/decisiones/gestor-dependencias.md

```diff
@@ -0,0 +1,13 @@
+# Elección del gestor de dependencias
+
+La inmensa mayoría de los gestores de dependencias más usados para proyectos de JavaScript/TypeScript son compatibles entre sí y todos usan en general los mismos paquetes. Por tanto, y por experiencia propia, como criterio de selección del gestor de dependencias se va a tener en cuenta la velocidad a la que instala los paquetes y la facilidad de añadir/eliminar paquetes y gestionar sus veriones. Y como criterio secundario, pero también muy importante, es que se mantenga actualizado.
+
+Teniendo estos criterios en cuenta, como gestores de dependencias tenemos varias opciones conocidas:
+
+- El gestor de paquetes de Node.js conocido como __npm__: es uno de los gestores de paquetes más extendidos y más usados en proyectos de JS. Al ser uno de los gestores de paquetes/dependencias más usados nos proporciona una gran cantidad de paquetes de código abierto a nuestra disposición. Es de código abierto y sigue siendo actualizado a día de hoy (su ultima actualización al momento de escribir esto fue hace 5 días).
+
+- __yarn__: no lo conozco de primera mano pero si he leído sobre este gestor de paquetes. Según [este artículo](https://openwebinars.net/blog/que-es-yarn/) funciona igual que npm con la diferencia de que éste es más seguro y rápido y usa un comprobador de licencias sus paquetes, lo cual ayuda a saber exactamente qué estamos insalando. También sigue siendo actualizado, pero menos frecuentemente que npm por lo que he visto en su repositorio de github.
```

Comment:
¿De veras creéis que esta información, que yo tengo que revisar, me sirve para dilucidar si habéis seguido un procedimiento correcto para elegir un sistema u otro?